automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -401,9 +408,176 @@ def sync_capsule_tools_repos_to_upgrade(admin_password=None):
+        client in defined sequence
+
+    Rhevm upgrade Env Vars:
+
+    RHEV_CLIENT_CV
+        The CV name used in client subscription
+    RHEV_CLIENT_ENVIRONMENT
+        The environment name used in client subscription
+    RHEV_CLIENT_AK
+        The AK name used in client subscription
+    """
+    client_os = client_os.upper()
+    tools_repo = os.environ.get('TOOLS_URL_{}'.format(client_os))
+    if tools_repo is None:
+        print('The Tools Repo URL for {} is not provided '
+              'to perform Client Upgrade !'.format(client_os))
+        sys.exit(1)
+    cv_name, env_name, ak_name = [
+        os.environ.get(env_var)
+        for env_var in (
+            'RHEV_CLIENT_CV', 'RHEV_CLIENT_ENVIRONMENT', 'RHEV_CLIENT_AK')
+    ]
+    details = os.environ.get('CLIENT_SUBSCRIPTION')
+    if details is not None:
+        cv_name, env_name, ak_name = [
+            item.strip() for item in details.split(',')]
+    elif not all([cv_name, env_name, ak_name]):
+        print('Error! The CV, Env and AK details are not provided for Client'
+              'upgrade!')
+        sys.exit(1)
+    # Set hammer configuration
+    set_hammer_config()
+    # Create product capsule
+    hammer_product_create('tools6_latest', '1')
+    time.sleep(2)
+    # create repo
+    hammer_repository_create(
+        'tools6_latest_repo', '1', 'tools6_latest', tools_repo)
+    # Sync repos
+    hammer_repository_synchronize(
+        'tools6_latest_repo', '1', 'tools6_latest')
+    # Add repos to CV
+    hammer_content_view_add_repository(
+        cv_name, '1', 'tools6_latest', 'tools6_latest_repo')
+    # Publish cv
+    hammer_content_view_publish(cv_name, '1')
+    # Promote cv
+    lc_env_id = get_attribute_value(
+        hammer('lifecycle-environment list --organization-id 1 '
+               '--name {}'.format(env_name)), env_name, 'id')
+    cv_version_data = hammer(
+        'content-view version list --content-view {} '
+        '--organization-id 1'.format(cv_name))
+    latest_cv_ver = sorted([float(data['name'].split(
+        '{} '.format(cv_name))[1]) for data in cv_version_data]).pop()
+    cv_ver_id = get_attribute_value(cv_version_data, '{0} {1}'.format(
+        cv_name, latest_cv_ver), 'id')
+    hammer_content_view_promote_version(cv_name, cv_ver_id, lc_env_id, '1')
+    # Add new product subscriptions to AK
+    hammer_activation_key_add_subscription(ak_name, '1', 'tools6_latest')
+    # Add this latest tools repo to hosts to upgrade
+    for host in hosts:
+        attach_subscription_to_host('1', 'tools6_latest', host)
+
+
+def remove_all_docker_containers(only_running=True):
+    """Deletes docker containers from system forcefully
+
+    If only_running is set to true then only running containers will be deleted
+    else all running + stopped containers will be deleted
+
+    :param bool only_running: Whether to delete only running containers
+    """
+    if only_running:
```

Comment:
un('docker rm $(docker ps -q{}) -f'.format('a' if only_running else ''))

Or, since only_running is a boolean you can assume use plain if/else block instead of checking for true (this line) and false on elif not only_running a simple else does the same there.